html/wp-content/themes/ailurotech/views/partial/socialmedia.twig

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
This is too hardcoded, these icons can be any icons I put on WordPress. Make it dynamic rather than hardcode everything